pom.xml

```diff
@@ -45,7 +45,6 @@
         <module>web-client</module>
         <module>sikkerhet</module>
         <module>leader</module>
-        <module>prosessering</module>
```

Comment:
I denne er det nog flere dependencies som nå kan slettes, som kun brukes i prosessering

Reply:
Ok, I'll look through the dependencies of prosessering

Reply:
I did not find any dependencies that can be deleted. Do you have suggestions? @blommish

Reply:
@MengZhuNAV He mentions a couple of ones in the comment. They are in the dependecyManagement part of parent pom

Reply:
@stigebil Thank you I see. removed